### ISSUE TITLE
Lightning Capacitor - How it actually worked after 2.3 nerf

### ISSUE THREAD

**Issue by maingaming** [SunstriderEmu/BugTracker#1646]
**Item:** The Lightning Capacitor (ID 28785)

The tooltip from the nerfed 2.3 version with 2.5sec ICD states that an Electrical Charge cannot be gained more often than once every 2.5sec. However, as it often happens with Blizzard's tooltips they can be misleading or simply false which was also the case for TLC.

**Setup:**
A) Elemental Shaman spamming Lightning Bolts (2sec cast time) vs 1 training dummy.
B) Elemental Shaman spamming Chain Lightning on cooldown vs 3 training dummies.

**Current behaviour on server:**
A) 2 LB crits in a row **or** 1 LB crit that also triggered a Lightning Overload (Shaman Talent) which also crit. Only the first LB generates an Electrical Charge.
B) CL crits on 2 or 3 dummies. Only 1 EC is generated.

**How it should be:**
A) If 2 LBs crit in a row they will generate 2 ECs **unless** the first LB triggered the 3rd EC which triggered the TLC-Bolt. 1 LB+LO crit will generate 2 ECs and overcap (see B).
B) If CL crits on 2 or 3 dummies TLC will generate as many ECs as there were crits, fire a TLC-Bolt if it reached 3 ECs and will overcap to a new stack of EC. However, it can't proc again **after** it shot a TLC-Bolt for the next 2.5sec. This should also work the same for e.g. Mage Shatter combo (Frost Bolt into Frost Lance at the same time with both critting).

**Sources:**
https://www.wowhead.com/item=28785/the-lightning-capacitor#comments:id=192862
https://www.wowhead.com/item=28785/the-lightning-capacitor#comments:id=219453
https://www.wowhead.com/item=28785/the-lightning-capacitor#comments:id=246496
https://www.wowhead.com/item=28785/the-lightning-capacitor#comments:id=313901
https://www.wowhead.com/item=28785/the-lightning-capacitor#comments:id=402192
(There might be more but I wasn't able to find the old EJ thread.)

**TL;DR:** The 2.3 nerf was aimed at preventing multiple TLC-Bolts firing at the same time, not preventing the build-up of multiple stacks of the Electrical Charge buff. The stack counter on EC should rather be read as: 1/3, 2/3 or 3/3 of a full charge.

EDIT:
There is a reason why this trinket was ranked so highly:
https://web.archive.org/web/20081222121504/http://wiki.shadowpriest.com/index.php?title=SimulationCraft/Trinkets/Shaman

**Comment by Aeded** [comment]
Or you could just simply say that after The Lightning Capacitor procs a lightning bolt, you are not able to gain a charge for 2,5s. And outside of this situation there is no cooldown on gaining the charges.

**Comment by blblblaz** [comment]
confirmed, thanks - https://youtu.be/IB7bUNWTlh4?t=873

**Comment by justheal** [comment]
btw, there is the same problem with Arcane missiles (should work like lightning bolt with talent) or any AOE spells (should work like chain lightning).
trinket is trash when it have constant 2.5 cooldown for charges like now.

> Or you could just simply say that after The Lightning Capacitor procs a lightning bolt, you are not able to gain a charge for 2,5s. And outside of this situation there is no cooldown on gaining the charges.

**Comment by EnterLarry** [comment]
Is this being looked into still?

**Comment by Jerrythemagicdog** [comment]
Thanks, fixed with next update.
